lib/hcc-config.cmake.in

```diff
@@ -22,6 +24,16 @@ set_target_properties(hsa-runtime64 PROPERTIES
   INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
 )
 
+pkg_check_modules(LIBUNWIND REQUIRED libunwind)
+add_library(${LIBUNDWIND_LIBRARIES} INTERFACE IMPORTED)
```

Comment:
again. why? I have asked this several times, and the only reason to use imported target was for downstream builds. However, those don't use the smae cmake files. what am I missing?